src/main.ts

```diff
@@ -79,11 +79,7 @@ async function startApp() {
   try {
     // Create native window
     const appWindow = new AppWindow();
-    appWindow.onClose(() => {
```

Comment:
So we no longer quit the app when the app window is closed by this change? Is this intended?